pillar/edx/ansible_vars/cloud_deployment.sls

```diff
@@ -45,7 +45,11 @@
 {% elif environment == 'mitxpro-production' %}
 {% set efs_id = 'fs-68918b88' %}
 {% endif %}
-{% set edxapp_google_analytics_account = '' %}
+{% if environment == 'mitxpro-qa' %}
+  {% set edxapp_google_analytics_account = 'UA-5145472-40' %}
+  {% elif environment = 'mitxpro-production' %}
```

Comment:
There's already a conditional for environment specific data just above here. Might as well keep it in there.